python/cudf/cudf/tests/test_list.py

```diff
@@ -478,16 +495,20 @@ def test_index(data, scalar, expect):
             [["a", "b", "c"], None, [None, "d"]],
             2,
         ),
+        (
+            [["e", "s"], ["t", "w"]],
+            [5, 6],
+        ),
     ],
 )
-def test_index_invalid(data, scalar):
+def test_index_invalid(data, search_key):
```

Comment:
Can we add a test for the case where the search key is not the right length? e.g. `len(sr) != len(search_key)`